packages/gatsby-plugin-sitemap/README.md

```diff
@@ -54,7 +54,7 @@ The [`default config`](https://github.com/gatsbyjs/gatsby/blob/master/packages/g
 
 The options are as follows:
 
-- `output` (string = `/sitemap`) Folder path where sitemaps are stored.
+- `output` (string = `/`) Folder path where sitemaps are stored.
```

Comment:
The current PR code has the default path as `/sitemap` so I guess this should be reverted, no?